plugin.video.invidious/resources/language/Makefile

```diff
@@ -0,0 +1,4 @@
+# Update translations when the english master changes
+
+update-po:
```

Comment:
Please remove this file from the PR request.

Reply:
No problem. Why do you want to hide the recipe on how to update the translations, build a distribution zip and make releases?

Reply:
> Why do you want to hide the recipe on how to update the translations, build a distribution zip and make releases?

I guess they have their own way to do releases.
I always branch out and create PRs from the branch, and keep my own potential none desired custom scripts to my own master branch.

(Btw, I was the origin for the Makefile. I use it for maintaning addon releases in my own kodi repo.)